### ISSUE TITLE
Namespace parameter doesn't seem to be working

### ISSUE THREAD

**Issue by mattymillar** [10gen-labs/mongo-connector#217]
When I run mongo-connector (v5) without specifying a namespace solr indexing runs correctly

mongo-connector -m localhost:27017 -t http://localhost:8983/solr/test1 -d solr_doc_manager

However, if I add a namespace in there (starting from scratch) indexing doesn't run

mongo-connector -n 2.Assets -m localhost:27017 -t http://localhost:8983/solr/test1 -d solr_doc_manager

The connector runs, no errors occur, but nothing happens. Any ideas?

**Comment by mpvinaykumar** [comment]
hi Millar,

when you are running with the above name-space command, are you getting any error in mongod(mongo server) saying "authentication failed to  for db '2' ..."...?

**Comment by mattymillar** [comment]
Nope! No errors at all. I worked around the issue after reading this:

https://github.com/10gen-labs/mongo-connector/issues/165

It doesn't provide any answers to why the issue is occurring, but it gave me something to try. All I did  after reading this was edit a document in the specified namespace and then the whole namespace began indexing. After that first index I could then delete the index and reindex to my hearts content. This was the same for a few of the databases on my server.

Cheers
